jlibra-core/src/main/java/dev/jlibra/admissioncontrol/query/GetAccountState.java

```diff
@@ -2,8 +2,22 @@
 
 import org.immutables.value.Value;
 
+import com.google.protobuf.ByteString;
+
+import types.GetWithProof.GetAccountStateRequest;
+import types.GetWithProof.RequestItem;
+
 @Value.Immutable
 public interface GetAccountState {
 
     byte[] getAddress();
+
+    default RequestItem toGrpcObject() {
```

Comment:
What do you think about better name, eg. `toRequestItem` ?

Reply:
Yeah, that sounds better. That toGrpcObject remained there because I first started by creating an interface with that method and the idea was that all classes that can be converted to a grpc object would implement it, but I dropped the interface as I didn't see any use for it.
I'll change the names for all of them.

Reply:
Not sure. Actually we are creating a Grpc object here. Also there is a dependency in Grpc for serialization. `toGrpcObject` sounds very specific, but that's actually what happens here. `toRequestItem` would be more correct from business-functionality perspective, but I think it's obfuscating things. I understand toGrpc better.

Reply:
Yes, you also have a point. The conversion to grpc object is only required for the call to the libra service and not internally in jlibra so it is similar to the serialize() methods and could be named more generally. What do you @hczerpak think based on these comments?

Reply:
I've thought about this and I think keeping it as toGrpcObject() makes sense, all of these jlibra classes have their counterpart in the libra grpc api, and the jlibra object needs to be transformed to a grpc object before the actual call, so having the same method in all of these classes sounds right. Maybe having the interface for this would make it clearer.